index.html

```diff
@@ -72,10 +74,10 @@
 			<thead>
 				<tr>
 					<th>Name</th>
-					<th>Diagonal</th>
+					<th>Diagonal (inches)</th>
 					<th>Resolution</th>
-					<th>DPI</th>
-					<th>dppx</th>
+					<th><span title="Dots Per Inch">DPI</span></th>
```

Comment:
I've made the changes accordingly. (Sorry, I'm a bit under-experience with HTML).